tests/integration/plugins/test_signer.py

```diff
@@ -46,7 +46,8 @@ async def test_signer_plugin_capabilities(make_client: ClientFactory):
     anonymous_client = make_client(tuple())
     capability = (await anonymous_client.server_info())["capabilities"]["signer"]
     assert capability["group_check_enabled"]
-    assert capability["to_review_enabled"]
+    # TODO: this is disabled on dev
+    # assert capability["to_review_enabled"]
```

Comment:
I would suggest that we remove this assertion here, but use the `capability["to_review_enabled"]` to adjust the scope of the `test_signer_plugin_full_workflow` below (or skip it completely if it's too coupled with review enabled)